packages/tc-ui/src/pages/delete/__tests__/e2e-delete-record.cyp.js

```diff
@@ -1,32 +1,22 @@
-const {
-	code,
-	deleteConfirmText,
-} = require('../../../test-helpers/mainTypeData.json');
-const {
-	createType,
-	visitEditPage,
-	visitMainTypePage,
-	pickChild,
-	save,
-} = require('../../../test-helpers/cypress');
+const { executeQuery, dropFixtures } = require('../../../test-helpers/db');
 
+const namespace = 'e2e-page-delete';
+const code = `${namespace}-code`;
+const deleteConfirmText =
+	'Are you sure you wish to delete?\n\nUnless you created something by accident, a more appropriate action is usually to mark the record as inactive, either in the Is Active or Lifecycle Stage fields,';
 describe('End-to-end - delete record', () => {
-	beforeEach(() => {
-		cy.wrap(createType({ code, type: 'MainType' })).then(() => {
-			visitMainTypePage();
-		});
-	});
-
+	before(() => cy.wrap(dropFixtures(namespace)));
+	afterEach(() => cy.wrap(dropFixtures(namespace)));
 	it('shows a prompt message', () => {
-		cy.visit(`/MainType/${code}`, {
+		cy.wrap(
```

Comment:
Highlight #13 Previously a lot of fixtures were created _in_ cypress by navigating to a create page, saving the record etc. Now these are all created by using the `cy.wrap()` method to wrap direct db calls, which really speeds things up